blocks/contact-form/contact-form.js

```diff
@@ -20,30 +25,59 @@ export default function decorate(block) {
       div.classList.add('contact-form__field');
     }
 
+    // Check for field name and add a fitting input "type" attribute and assign variables
+    let firstName;
+    let lastName;
+    let email;
+    let birthDate;
+    let termsOfService;
     // Add inputs with fitting "type" attributes to every field except for the title
     if (!fieldName.includes('title')) {
       // Create input element
       const input = document.createElement('input');
 
-      // Check for field name and add a fitting input "type" attribute
       if (fieldName.includes('name')) {
         input.setAttribute('type', 'text');
       }
+      if (fieldName.includes('first name')) {
```

Comment:
do not use whitespace in id-type fields (like `fieldName` in this case). a JS-type convention would be to use `camelCase`, similar to the variable you're defining here.